parseresults.py

```diff
@@ -43,8 +47,7 @@
         else:
             failures += 1
 
-
-print("Amount of steps = " + str(len(testResults)))
+print("Amount of steps = " + str(len(testResults)) + " (Claimed " + numTests + ")" )
 
```

Comment:
Like this?
```print("Amount of steps = {} (Claimed {})".format(len(testResults), numTests))```

It's a bit closer to c-style, I like the fact that it allows you to easier see how the string itself will look. But then seeing which variable occupies which space is tricky if there are several of them.

Or like this?
```print("Amount of steps = %i (Claimed %s)" % (len(testResults), numTests))```
it says here that it is depreciated in python 3.1 http://www.pythonforbeginners.com/concatenation/string-concatenation-and-formatting-in-python so I'd rather not use it.